packages/hydrogen/src/cart/cart-queries.ts

```diff
@@ -0,0 +1,364 @@
+const USER_ERROR_FRAGMENT = `#graphql
+  fragment ErrorFragment on CartUserError {
+    message
+    field
+    code
+  }
+`;
+
+export const MINIMAL_CART_FRAGMENT = `#graphql
+  fragment CartFragment on Cart {
+    id
+    totalQuantity
+  }
+`;
```

Comment:
In codegen, generated types for fragments are automatically appended `...Fragment` to their name.
Can we find a different pattern to name our fragments here? It's also clear they are fragments since GraphQL forces that keyword, so what about using "Selection"?

<img width="747" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/4542997a-09df-4d55-9242-6f2c702db298">

Or, maybe just describing what they are actually used for (from the user perspective):
- `fragment CartApiQuerySelection` / `fragment CartApiQuery`
- `fragment CartApiMutationSelection` / `fragment CartApiMutation`

Thoughts?